packages/ui/src/components/alert/Alert.test.tsx

```diff
@@ -16,7 +16,7 @@ describe('Alert', () => {
       </Alert.Container>
     )
 
-    expect(screen.getByText(title)).toBeInTheDocument()
-    expect(screen.getByText(description)).toBeInTheDocument()
+    expect(await screen.findByRole('heading', { name: title })).toBeInTheDocument()
```

Comment:
Best to always opt for `ByRole` when you can and only fall back to `ByText` when there's nothing else to grab on to.